pkgs/development/python-modules/xhtml2pdf/default.nix

```diff
@@ -0,0 +1,33 @@
+{lib, fetchPypi, buildPythonPackage, html5lib, httplib2, nose, pillow, pypdf2, reportlab}:
+
+let
+  #xhtml2pdf specifically requires version "1.0b10" of html5lib
+  html5 = html5lib.overrideAttrs( oldAttrs: rec{
```

Comment:
We permit only a single version within `python-packages.nix`, so no overriding here.

Reply:
Considering sasview is the only consumer of this package, I suggest moving the expression there.